packages/dotcom-ui-footer/src/__stories__/story.tsx

```diff
@@ -1,37 +1,45 @@
 import React from 'react'
 import { OnReady } from '../../../../.storybook/components/OnReady'
 
-import { storiesOf } from '@storybook/react'
-import { withKnobs, radios } from '@storybook/addon-knobs'
 import storyData from './story-data'
 
 import * as footer from '../../browser.js'
 import { Footer, LegalFooter } from '../../src'
 import '../../styles.scss'
 
-const toggleTheme = () =>
-  radios(
-    'Theme',
-    {
-      dark: 'dark',
-      light: 'light'
-    },
-    'dark'
+export default {
+  title: 'FT / Footer',
+  args: { theme: 'dark' },
+  argTypes: {
```

Comment:
Question (I've not used storybook before): 

As the components are written in typescript, do we need to specifically add `argTypes`? can we get that for free?

Reply:
It seems to default to a text input and not infer types from TypeScript so I dont think so unfortunately.